### ISSUE TITLE
rshell --quiet outputs a fullstop on newly inserted board

### ISSUE THREAD

**Issue by rhubarbdog** [dhylands/rshell#101]
I'm using rshell from bash. On a newly plugged in board with the `--quiet` option a fullstop is output to stdout.
I plug in a  pyboard, from bash i type `rshell --quiet ls /sd` and get the following 
```
.boot.py main.py
```
i type `rshell --quiet ls /sd` again and the fullstop isn't there
```
boot.py main.py
```

**Comment by rhubarbdog** [comment]
As of version 0.0.24 this text is always output
```
Trying to open serial port /dev/ttyACM0
Serial port opened successfully True
```

**Comment by dhylands** [comment]
I just released 0.0.25 which includes this fix
